druid/src/core.rs

```diff
@@ -798,6 +984,7 @@ impl<'a, 'b> UpdateCtx<'a, 'b> {
     /// Note: For the most part we're trying to migrate `WindowHandle`
     /// functionality to `WinCtx`, but the update flow is the exception, as
     /// it's shared across multiple windows.
+    //TODO: can we delete this? where is it used?
```

Comment:
If memory serves, it's a vestige of the ext_event mechanism, where we use it to get an `IdleHandle` that can then be used to submit an ext_event. That all will be replaced with something based on `Command`, so yes, I think now might be a good time to delete it. We'll restore the functionality as #471.